src/pages/DataSamplesPage.tsx

```diff
@@ -75,6 +75,7 @@ const DataSamplesPage = () => {
               {model ? (
                 <Button
                   onClick={handleNavigateToModel}
+                  className={tourElClassname.trainModelButton}
```

Comment:
We don't adapt the text in this case... but it seems hard to do well in terms of copy and important to talk about "Train model" nonetheless.